scripts/actions/utils/serialization-helpers.js

```diff
@@ -6,19 +6,28 @@ const toMDAST = require('remark-parse');
 const remarkMdx = require('remark-mdx');
 const remarkMdxjs = require('remark-mdxjs');
 const unified = require('unified');
+const visit = require('unist-util-visit');
+
+const removeParagraphs = () => (tree) => {
```

Comment:
why was the removeParagraphs added in?

Reply:
This is only used in the case of serializing a JSX expression for a prop. You can see the effect of the change here: https://github.com/newrelic/docs-website/pull/593/files#diff-190fa2e1660b11a6496776193cc476619e6b9ae0776db130bc3d7f157e50cd82R203-R205

Notice the `p` tag wrapped around the children there. This results in this:

```js
<Collapser
  title={<><InlineCode>code</InlineCode> API</>}
>
```

getting turned into this:

```js
<Collapser
  title={<>
    <InlineCode>code</InlineCode> AP
  </>}
```

Figured its best to try and keep the formatting as consistent as possible, so by removing the paragraphs, we can get rid of the additional formatting here.

Reply:
gotcha! thank you for the explanation